node/server.js

```diff
@@ -8582,37 +8584,38 @@ function init_io() {
 					resend(target, "u+cid");
 				}
 			} else if (data.name == "cburst") {
-				var hit = {};
-				var times = 0;
+				var targeted = {};
 				var attack = null;
 				var c_resolve = null;
 				consume_mp(player, gSkill.mp);
 				player.first_burst = true;
 				player.halt = true;
 				if (is_array(data.targets)) {
-					data.targets.forEach(function (t) {
-						// console.log(id);
-						var id = t[0];
-						var mp = max(0, parseInt(t[1]) || 0);
-						if (player.mp < 20 || times > 16 || !mp) {
-							return;
+					// Only look at the first 16 targets in the array if more are provided
+					for (const t of data.targets.slice(0, 16)) {
+						const id = t[0];
+						if (targeted[id]) {
```

Comment:
Add a clarifying comment that this is to prevent attacking the same entity twice